src/BenchmarkDotNet/Columns/Column.cs

```diff
@@ -55,6 +55,7 @@ public static class Column
         public const string CompletedWorkItems = "Completed Work Items";
         public const string LockContentions = "Lock Contentions";
         public const string CodeSize = "Code Size";
+        public const string ExceptionFrequency = "Exception frequency";
```

Comment:
I've run following command:

```cmd
cd samples\BenchmarkDotNet.Samples
dotnet run -c Release -f net6.0 --filter *IntroExceptionDiagnoser* --memory
```

And I got:

|                 Method | Exception frequency |   Gen0 | Allocated |
|----------------------- |--------------------:|-------:|----------:|
| ThrowExceptionRandomly |              0.6005 | 0.0229 |     207 B |
|              DoNothing |                   - |      - |         - |

I am wondering whether we should change "Exception frequency" to just "Exceptions" to save some space in the produced output.
```suggestion
        public const string Exceptions = "Exceptions";
```